packages/react-sdk/src/page-settings-meta.tsx

```diff
@@ -137,5 +137,12 @@ export const PageSettingsMeta = ({
 
   metas.push(...pageMeta.custom);
 
+  const isTwitterCardSizeExists = pageMeta.custom.some(
```

Comment:
so this will only work if twitter:card meta is added? why not always have it if there is og image provided?

Reply:
It's jst a check to see if users added `twitter:card` and skips if there is any. Or else, adds the `summary_large_image`. Just to give preference to whatever it is added. Renamed it for better red-ability.